deployment/docker-inasafe-headless/Dockerfile

```diff
@@ -49,7 +49,8 @@ RUN pip install -r /REQUIREMENTS-headless.txt
 RUN pip install -r /REQUIREMENTS-realtime.txt
 # Copy ubuntu fonts
 RUN apt-get -y install wget unzip
-RUN wget -c http://font.ubuntu.com/download/ubuntu-font-family-0.83.zip
+RUN wget -c https://assets.ubuntu.com/v1/fad7939b-ubuntu-font-family-0.83.zip
+RUN mv fad7939b-ubuntu-font-family-0.83.zip ubuntu-font-family-0.83.zip
```

Comment:
I also encountered this problem. We should probably saved the cache in this repo.